google/cloud/storage/testbench/requirements.txt

```diff
@@ -1,4 +1,4 @@
-crc32c==2.0
-Flask==1.1.1
+crc32c==2.1
+Flask==1.1.2
```

Comment:
All the other edits used lower-case "flask".  Should we be consistent?

Reply:
Good idea, thanks, and done.